agreementbot/api.go

```diff
@@ -641,8 +663,23 @@ func NewHorizonAgbot(id string, org string) *HorizonAgbot {
 	}
 }
 
-func getAgbotInfo(config *config.HorizonConfig) {
+type HorizonAgbotConfig struct {
+	InMemoryConfig   config.AGConfig `json:"In_Memory_Config"`
```

Comment:
Minor tweak. We try to use camel case, so the json field names should be InMemoryConfig and FileSystemConfig.